pkg/types/ccipocr3/config.go

```diff
@@ -0,0 +1,77 @@
+package ccipocr3
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/smartcontractkit/libocr/commontypes"
+	"github.com/smartcontractkit/libocr/offchainreporting2plus/types"
+)
+
+type CommitPluginConfig struct {
+	// DestChain is the ccip destination chain configured for the commit plugin DON.
+	DestChain ChainSelector `json:"destChain"`
+
+	// FChain defines the FChain value for each chain. FChain is used while forming consensus based on the observations.
+	FChain map[ChainSelector]int `json:"fChain"`
+
+	// ObserverInfo is a map of oracle IDs to ObserverInfo.
+	ObserverInfo map[commontypes.OracleID]ObserverInfo `json:"observerInfo"`
+
+	// PricedTokens is a list of tokens that we want to submit price updates for.
+	PricedTokens []types.Account `json:"pricedTokens"`
+
+	// TokenPricesObserver indicates that the node can observe token prices.
+	TokenPricesObserver bool `json:"tokenPricesObserver"`
+
+	// NewMsgScanBatchSize is the number of max new messages to scan, typically set to 256.
+	NewMsgScanBatchSize int `json:"newMsgScanBatchSize"`
+}
+
+func (c CommitPluginConfig) Validate() error {
```

Comment:
Where is this called from?

Reply:
from plugin

Reply:
Do you have a link? I couldn't find it in the PR diff.

Should it have uses in this module? At least some simple tests?

Reply:
https://github.com/smartcontractkit/ccip/blob/46fd2d428da707bb56d39222814dedd4f027e29b/core/services/ocr3/plugins/ccip/commit/plugin.go#L178

Reply:
added tests

Reply:
good catch